src/jit/standalone/CMakeLists.txt

```diff
@@ -1,8 +1,14 @@
 project(ryujit)
-add_definitions(-DFEATURE_NO_HOST)
-add_definitions(-DSELF_NO_HOST)
-add_definitions(-DFEATURE_READYTORUN_COMPILER)
-add_definitions(-DFEATURE_REF_ZERO_OFFSET_ALLOWED)
+
+# We will need to build ryujit as an altjit for targets we
+# will be bringing up the jit in.
```

Comment:
The ryujit .dll packaged into RyuJIT nuget package should always be the proper RyuJIT, otherwise it will be source of endless confusion. If you would like to do mixing and matching for CoreRT, it should be done by JIT .dll that implements this policy. @russellhadley was talking about having one for mixing and matching ryujit with llilc.